docs/content/administration/config-cheat-sheet.en-us.md

```diff
@@ -815,6 +815,11 @@ Default templates for project boards:
 - `PROJECT_BOARD_BASIC_KANBAN_TYPE`: **To Do, In Progress, Done**
 - `PROJECT_BOARD_BUG_TRIAGE_TYPE`: **Needs Triage, High Priority, Low Priority, Closed**
 
+## Project Automation (`project.automation`)
+
+- `ENABLED`: **false**: Whether automation is enabled for projects.
+- `MAX_RULE_PER_PROJECT`: **25**: Maximum number of automation rules per project.
```

Comment:
Do we even need such a setting?
I think it is much easier if we implement it at first without any `max amount of rules`.

Reply:
Especially as you then wouldn't need to **hardcode** the number of rules.
You should instead loop over all keys in this section instead.

Reply:
The intent of this setting was to limit the maximum amount of rules considered at runtime, but I understand that this causes confusion at this stage.

I agree that this can be removed for the time being, as it can probably be reintroduced at a later time when automation rules can be added via the project settings in the GUI.